src/main/java/org/commcare/cases/instance/StorageInstanceTreeElement.java

```diff
@@ -273,7 +273,7 @@ protected Model getElement(int recordId) {
     }
 
     protected Model getModelTemplate() {
-        return storage.read(1);
```

Comment:
It did indeed cause serious issues. This is just tests for the bugfix, which I applied in a separate PR. There are a couple of issues with the new fixture format which need to be addressed (Nestle submitted them as feedback)